sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -238,11 +244,15 @@ def __init__(self, message, *args, **kwargs):
         super(AzureError, self).__init__(self.message, *args)
 
     def raise_with_traceback(self):
+        """Raise the exception with the existing traceback.
+
+        This method is deprecated as we don't support Python 2 anymore. Use raise/from instead.
```

Comment:
We can use the [deprecated](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) directive here :)
```suggestion
        .. deprecated:: 1.22.0
           This method is deprecated as we don't support Python 2 anymore. Use raise/from instead.
```